autoload/fruzzy/ctrlp.vim

```diff
@@ -0,0 +1,52 @@
+"
+" Sources:
+" - https://github.com/nixprime/cpsm
+" - https://sourcegraph.com/github.com/nixprime/cpsm@master/-/blob/autoload/cpsm.py#L27
+" - https://github.com/raghur/fruzzy/issues/1
+"
+if has('python3')
+  let pyfile = 'py3file '
+  let py = 'py3eval'
+elseif has('python')
+  let pyfile = 'pyfile '
+  let py = 'pyeval'
+else
+  echom "Can't load fruzzy_ctrlp!"
+  finish
+endif
+
+let s:script_dir = escape(expand('<sfile>:p:h'), '\')
+let s:fruzzy_dir = fnamemodify(s:script_dir, ':h:h') . 'rplugin/python3'
+
+execute pyfile . s:script_dir . '/ctrlp.py'
```

Comment:
You could keep the ctrlp.py script in <plugin>/python3 and in the py script, figure out the path to fruzzy.py, add it to sys.path and then import it. Then your vimscript matcher function can just be `import fruzzy_ctrlp; call yourwrapperfunc(args)`